relayer/src/contracts/azero.rs

```diff
@@ -35,6 +35,31 @@ pub enum AzeroContractError {
     MissingOrInvalidField(String),
 }
 
+pub struct AdvisoryInstance {
+    pub contract: ContractInstance,
+    pub address: AccountId,
+    pub transcoder: ContractMessageTranscoder,
```

Comment:
```suggestion
```

Reply:
something went wrong with this comment?

Reply:
Why do you need this fields in `AdvisoryInstance`? I don't see them used anywhere

Reply:
I don't need the transcoder now that the event subscription is gone, but the address is needed (for logging)